avidemux_ru.ts

```diff
@@ -2767,7 +2767,7 @@ Is it running ?</source>
     <message>
         <location line="+4"/>
         <source>Enable OpenGL display in filter preview dialogs</source>
-        <translation type="unfinished"></translation>
+        <translation>Включить отображение OpenGL в окне предпросмотра фильтров</translation>
```

Comment:
The new feature uses OpenGL to display stuff, not stuff to display OpenGL. Is it possible that the translation can be (mis)understood as the latter?

Reply:
Reworded for better meaning.